packages/kbn-dev-proc-runner/src/proc_runner.ts

```diff
@@ -131,10 +131,13 @@ export class ProcRunner {
         );
       }
 
-      if (wait === true) {
+      if (wait) {
```

Comment:
Let's revert this, to see if it's causing the timeouts:
```suggestion
      if (wait === true) {
```